cmsl1t/filters/luminosity.py

```diff
@@ -0,0 +1,45 @@
+import json
```

Comment:
Might it be worth using yaml loader here instead of json?  Since json is a subset of yaml, this might allow us to write more easily custom lumi-section filters?